cli/runtime/config/envs.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"github.com/pkg/errors"
+
+	configapi "github.com/vmware-tanzu/tanzu-framework/cli/runtime/apis/config/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/config/nodeutils"
+
+	"gopkg.in/yaml.v3"
+)
+
+func GetAllEnvs() (map[string]string, error) {
```

Comment:
nit: it would be good if we have documentation for this function

Reply:
added docs